README.rst

```diff
@@ -20,11 +20,7 @@ This service is configured with an in-memory database simply to make Django happ
 Operators
 =========
 
-This is intended to be run as a fully internal service with no database or admin frontend, with the LMS and CMS making calls to it unauthenticated. **It should not be callable directly from the internet.**
-
-While the service uses AppArmor as a first (and really, only) line of defense, it should also be deployed in a way that does not allow direct connections to other IDAs or internal services within the Open edX deployment. In the ideal situation, networking would be set up to only allow outbound connections to a predetermined set of IPs or domains.
-
-After any significant change to security settings, consider running the tests in ``./api_tests/``. These are run manually against a deployed service and can check for various sandbox weaknesses.
```

Comment:
I'd note here something about the important of properly securing this service, and sending people to the deployment doc for details.